Runtime/Components/Cinemachine3rdPersonFollow.cs

```diff
@@ -239,19 +237,19 @@ Quaternion GetHeading(Vector3 targetForward, Vector3 up)
             Vector3 root, Quaternion targetRot, Quaternion heading, 
             out Vector3 shoulder, out Vector3 hand)
         {
-            var shoulderPivotReflected = Vector3.Reflect(ShoulderOffset, Vector3.right);
-            var shoulderOffset = Vector3.Lerp(shoulderPivotReflected, ShoulderOffset, CameraSide);
+            var shoulderOffset = ShoulderOffset;
+            shoulderOffset.x = Mathf.Lerp(-shoulderOffset.x, shoulderOffset.x, CameraSide);
```

Comment:
Changed this from `shoulderOffset.x = Mathf.Lerp(-ShoulderOffset.x, ShoulderOffset.x, CameraSide);`